README.md

```diff
@@ -105,7 +105,7 @@ If you wish to see the raw output from our API in this response, supply  the `--
 
 #### Create a new version
 ```sh
-rdme versions:create <version>
+rdme versions:create --version={project-version}
```

Comment:
Technically both of these are correct since `version` is configured as a `defaultOption` on these commands.

Reply:
Checkout the issue I linked, they were not being parsed in the same manner due to how the cli package handles booleans interacting with multiflag. Since they've been shifted to strings, it seems to be working as you mentioned. I'll reflect this :)